autoware_launch/launch/autoware.launch.xml

```diff
@@ -51,12 +51,14 @@
 
   <!-- Localization -->
   <include file="$(find-pkg-share localization_launch)/launch/localization.launch.xml">
+    <arg name="input_sensor_points_topic" value="/sensing/lidar/pointcloud" />
```

Comment:
can you change "input/sensing/pointcloud"?

Reply:
This line has removed, should I still fix it?

Reply:
@yukkysaito Could you comment?

Reply:
ping @yukkysaito

Reply:
https://github.com/tier4/autoware_launch/pull/215#issuecomment-1062810005